src/reanimated2/core.js

```diff
@@ -234,6 +235,16 @@ export const runOnJS = (fun) => {
   }
 };
 
+export function createAnimatedPropAdapter(adapter, nativeProps) {
+  const nativePropsToAdd = {};
+  // eslint-disable-next-line
+  nativeProps?.forEach((prop) => {
```

Comment:
Same as above, could you write a short comment on why this is ignored?

Reply:
fixed 65f7cfb